datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/handler/DatalakeFullBackupWaitHandler.java

```diff
@@ -56,19 +64,25 @@ protected Selectable doAccept(HandlerEvent<DatalakeFullBackupWaitRequest> event)
         String userId = request.getUserId();
         Selectable response;
         try {
+            PollingConfig pollingConfig;
             LOGGER.info("Start polling datalake full backup status for id: {}", sdxId);
-            PollingConfig pollingConfig = new PollingConfig(sleepTimeInSec, TimeUnit.SECONDS, durationInMinutes,
-                TimeUnit.MINUTES);
+            if (entitlementService.isLongTimeBackupEnabled(ThreadBasedUserCrnProvider.getAccountId())) {
+                pollingConfig = new PollingConfig(sleepTimeInSec, TimeUnit.SECONDS, longDurationInMinutes,
+                        TimeUnit.MINUTES);
+            } else {
+                pollingConfig = new PollingConfig(sleepTimeInSec, TimeUnit.SECONDS, durationInMinutes,
+                        TimeUnit.MINUTES);
+            }
```

Comment:
How about simplifying the change by adding below? The rest of the code need not change.
```
int durationInMinutes 
 = (entitlementService.isLongTimeBackupEnabled(ThreadBasedUserCrnProvider.getAccountId()) ? longDurationInMinutes : durationInMinutes;
```

Reply:
modified

Reply:
modified